committeeoversightapp/models.py

```diff
@@ -297,6 +321,13 @@ def percent_passed(self):
 
         return cap_100(percent_passed)
 
+    @property
```

Comment:
Does this offer any advantage over `if congress.footnote`?

Reply:
Good point 🤦🏻‍♀️